### DISCUSSION TITLE
Remover botão fixo do PoModal

### DISCUSSION THREAD

**Discussion by Lourencobrah** [po-ui/po-angular#1522] [Tire suas dúvidas]
Olá devs!

Tenho uma dúvida em relação ao **Modal**, nele existe um botão que vem por padrão com o nome "Fechar", existe alguma maneira de eu remover esse botão e deixar apenas a função de fechar clicando fora do modal?
Estou em uma situação onde o botão não serve e desejo remove-lo!

Att.

**Comment by jcorrea97** [comment]
@Lourencobrah, você consegue tirar esse botão de fechar passando um po-modal-footer vazio, por exemplo:

```
<po-modal #modal p-title="PO Modal" [p-hide-close]="true" [p-click-out]="true">
    Meu modal
    <po-modal-footer>
    </po-modal-footer>

</po-modal>

<po-button p-label="Open modal" (p-click)="modal.open()"> </po-button>
```
